### PR TITLE
Tweak how ast.pyi imports typing

### PR DESCRIPTION
**Pull request by gvanrossum**
When we import typeshed internally at Dropbox, somehow the fact that
these files are all stubs gets lost (it's a long story...).  This
causes errors like this:
```
  .../stdlib/2/ast.pyi:6: error: Name 'typing' already defined (by an import)
```
The quickest way around this is to rename the import to `_typing`.

### CODE REVIEW

File: stdlib/2/ast.pyi [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Python 2.7 ast
 
-import typing
+import typing as _typing
 from typing import Any, Iterator, Union
```
Comment (JelleZijlstra) [inline stdlib/2/ast.pyi]: Could you just add Tuple to this from import instead?

**Comment by srittau** [comment]
Now conflicts with master due to PR #2667, which added `Optional` to the list of imports. Should be trivial to fix.

File: stdlib/2/ast.pyi [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Python 2.7 ast
 
-import typing
+import typing as _typing
```
Comment (srittau) [inline stdlib/2/ast.pyi]: Could you add a small comment explaining this problem here (and in the Python 3 stub) for future generations?

Comment (gvanrossum) [reply stdlib/2/ast.pyi]: Done.

**Pull request merged**